{
  try {
    debug=Messages.getString("FilterRows.0");
    boolean ret=meta.getCondition().evaluate(row);
    debug=Messages.getString("FilterRows.Debug.AfterEvaluateCondition");
    return ret;
  }
 catch (  Exception e) {
    String message=Messages.getString("FilterRows.Exception.UnexpectedErrorFoundInEvaluationFuction") + e.toString();
    logError(message);
    logError(Messages.getString("FilterRows.Log.ErrorOccurredForRow") + row);
    logError(Const.getStackTracker(e));
    throw new KettleException(message,e);
  }
}
